src/uct/ib/base/ib_iface.c

```diff
@@ -82,13 +82,21 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation), UCS_CONFIG_TYPE_UINT},
 
 #if HAVE_DECL_IBV_EXP_CQ_MODERATION
-  {"TX_CQ_MODERATION_COUNT", "-1",
+  {"TX_CQ_EVENT_MODERATION_COUNT", "65535", /* MAX UINT16 */
    "Number of send WQEs for which event is requested.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation_count), UCS_CONFIG_TYPE_INT},
+   ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation_count), UCS_CONFIG_TYPE_UINT},
 
-  {"TX_CQ_MODERATION_PERIOD", "-1",
-   "Time period to generate event.",
-   ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation_period), UCS_CONFIG_TYPE_INT},
+  {"TX_CQ_EVENT_MODERATION_PERIOD", "65535", /* MAX UINT16 */
```

Comment:
need to specify this parameter in time units and provide more info in the documentation

Reply:
actually there is no time units in docs, I assume it is usec (at least experiments demonstrated unit)